static/app/views/organizationGroupDetails/actions/subscribeAction.tsx

```diff
@@ -22,7 +22,7 @@ function SubscribeAction({disabled, group, onClick}: Props) {
       title={getSubscriptionReason(group, true)}
       tooltipProps={{delay: 300}}
       priority={group.isSubscribed ? 'primary' : 'default'}
-      size="zero"
+      size="xsmall"
```

Comment:
Not sure why this was set to "zero" to begin with.

Before:
<img width="191" alt="subscribeAction--b" src="https://user-images.githubusercontent.com/44172267/150042465-b477ef14-5420-4c88-87da-6c9250511bb7.png">

After:
<img width="191" alt="subscribeAction--a" src="https://user-images.githubusercontent.com/44172267/150042479-d2c7d80a-d315-483f-b04c-e070f8b90ec2.png">